src/core/transaction/TransactionSender.ts

```diff
@@ -11,21 +11,22 @@ export class TransactionSender implements ISender {
   async send(config: PaymentConfiguration, paymentProcess: PaymentProcess): Promise<void> {
     const { payoutHash, senderKeys } = config;
 
-    const { blocks, payouts } = paymentProcess;
+    const { blocks, payoutTransactions } = paymentProcess;
 
-    const calculatedHash = hash(paymentProcess.storePayout, { algorithm: 'sha1' });
+    //const calculatedHash = hash(paymentProcess.payoutDetails, { algorithm: 'sha1' });
+    const calculatedHash = createHash('sha1').update(paymentProcess.payoutDetails.toString()).digest('hex');
```

Comment:
This is almost definitely wrong, need deep object inspection here. May want to revert.

Reply:
changed to stringify - resolved.